core/templates/pages/skill-editor-page/editor-tab/skill-misconceptions-editor/misconception-editor.component.ts

```diff
@@ -124,7 +125,6 @@ export class MisconceptionEditorComponent implements OnInit {
         this.misconception.getId(),
         this.nameMemento,
         this.container.misconceptionName);
-      this.nameMemento = null;
```

Comment:
Making momento null is not required.

Earlier:
previous value->null->new value

Now
previous value-> new value.